ruby_on_rails/mailers_advanced_topics/actioncable_lesson.md

```diff
@@ -91,7 +91,7 @@ Lastly, to add the navbar functionality we can just add it straight into the mai
     <div class="navbar-item">
       <div class="buttons">
         <% if current_user %>
-          <%= link_to "Logout", destroy_user_session_path, method: :delete, class: 'button is-link' %>
+          <%= link_to "Logout", destroy_user_session_path, data: { turbo-method: :delete }, class: 'button is-link' %>
```

Comment:
Nit: Small typo, we need an underscore here, a dash in a symbol will raise an error:

```suggestion
          <%= link_to "Logout", destroy_user_session_path, data: { turbo_method: :delete }, class: 'button is-link' %>
```

Reply:
Should have checked carefully before I commit last time. Thanks, should be good this time